### ISSUE TITLE
User avatar popup won't show the email for the first time

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3975]
### Expected behavior
Should display the email on the popup

### Actual behavior
User avatar popup won't show the email for the first time

### Steps to reproduce the problem

- Login as Customer
- Go to Grid view
- In the Project listing page hover over the current user avatar (You don't see email in the first time)
- Switch the tab to 'Reports'
- Switch back to 'All Project'
- In the Project listing page hover over the current user avatar (Now you can see the email)

### Screenshot/screencast

https://monosnap.com/file/uzQ3l0GM8qPGF7axAPTzS7cMbZz6Da

#### Environment
**- OS:** HP 15-dw1xxx 15.6 inch; Window 10
**- Browser (w/version):** Chrome 81.0.4044.129
**- User role (client, copilot or manager):**  Customer
**- Account used:**  `TopConnCustomer`

**Comment by vikasrohit** [comment]
It should not have shown the email for the non admin at all. @maxceem I think you have it fixed in patch release branch, right? If yes, I think we need to make it to production soon. I would do it tomorrow if QA passes for this.

**Comment by maxceem** [comment]
> It should not have shown the email for the non admin at all. 

@vikasrohit That's true except of user himself. User can see his own email. So as per this issue user sees his email inconsistently.

I think it's a very minor issue.

**Comment by vikasrohit** [comment]
ohk.

**Comment by maxceem** [comment]
I can reproduce it with `TopConnCustomer` user, but cannot reproduce it with `pshah_customer` in some reason.

**Comment by codejamtc** [comment]
@maxceem Yes, correct
